plugins/techdocs-node/api-report.md

```diff
@@ -85,6 +86,15 @@ export const getLocationForEntity: (
   scmIntegration: ScmIntegrationRegistry,
 ) => ParsedLocationAnnotation;
 
+// @public
+export const getMkdocsYml: (
+  inputDir: string,
+  siteName?: string,
```

Comment:
Maybe make this an optional options? I think we can extend on this additional parameters in the future.

Reply:
Yes, good call, also done 👍